app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
I would prefer that all cache expiration stay together in the `after_create` method, so that this kind of race condition is not possible even theoretically. 👍